src/main.js

```diff
@@ -130,7 +131,7 @@ function showSavedCovers() {
   savedCoversSection.innerHTML = '';
   for (var i = 0; i < savedCovers.length; i++) {
     savedCoversSection.innerHTML += `
-    <article class="mini-cover">
+    <article class="mini-cover" id=${savedCovers[i].id}>
```

Comment:
Nice! I like the effective use of a template to refresh the cover section, as opposed to trying to edit everything inline.